os/arch/arm/src/tiva/lm3s_ethernet.c

```diff
@@ -1165,9 +1093,7 @@ static int tiva_ifup(struct net_driver_s *dev)
 	 */
 
 	nlldbg("Waiting for link\n");
```

Comment:
May I know why this is removed? Also, if it is not necessary, maybe we should not have the nlldbg statement above.